### PR TITLE
mailtrap implemented

### PR DESCRIPTION
**Pull request by nicoopalenque**
# Sumary

- When you create a rent automatically send an email to the user
- Config body email
- Config credentials in secrets.yml and dotenv

# Card

[(2) Mandar email al usuario cuando se crea un Rent](https://trello.com/c/xZDs81uP)

# Screenshot

Captura de Pantalla 2021-07-01 a la(s) 23 22 54

### CODE REVIEW

File: app/mailers/rent_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class RentMailer < ApplicationMailer
+  def rent_email
+    @rent = params[:rent]
+    mail(
+      to: @rent.user.email,
+      subject: "Tanks for rent #{@rent.user.first_name} #{@rent.user.last_name}"
```
Comment (redwarewolf) [inline app/mailers/rent_mailer.rb]: "Thanks for your rent #{@rent.user.first_name} #{@rent.user.last_name}!"
There is a typo :P

File: app/views/rent_mailer/rent_email.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<ul>
+  <li>Title: <%= @rent.book.title %></li>
+  <li>Genre: <%= @rent.book.gender %></li>
+  <li>Author: <%= @rent.book.author %></li>
+  <li>Year: <%= @rent.book.year %></li>
+  <li>Editor: <%= @rent.book.editor %></li>
+</ul>
+
+<p>Rent start: <%= @rent.rent_start.strftime('%d-%m-%Y') %></p>
+<p>Rent end: <%= @rent.rent_end.strftime('%d-%m-%Y') %></p>
```
Comment (redwarewolf) [inline app/views/rent_mailer/rent_email.html.erb:15]: Oh !"#%...HTML, @blacksam07 I live this one to you xD

File: app/workers/hard_worker.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class HardWorker
+  include Sidekiq::Worker
+
+  def perform(*args)
+    # Do something
```
Comment (redwarewolf) [inline app/workers/hard_worker.rb]: 👀

File: spec/mailers/rent_mailer_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe RentMailer, type: :mailer do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (redwarewolf) [inline spec/mailers/rent_mailer_spec.rb]: delete the file.

File: spec/workers/hard_worker_spec.rb [hunk 5]
```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+RSpec.describe HardWorker, type: :worker do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (redwarewolf) [inline spec/workers/hard_worker_spec.rb]: delete the file.

**Review by redwarewolf — CHANGES_REQUESTED** [review]

File: app/views/rent_mailer/rent_email.html.erb (hunk 2 above)
Comment (blacksam07) [reply app/views/rent_mailer/rent_email.html.erb:15]: is the beautiful erb

**Review by blacksam07 — APPROVED** [review]

**Pull request merged**